projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/recommendations.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * Internal dependencies
+ */
+import WpPage from '../wp-page';
+
+const PERSONAL_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(1) input';
+const BUSINESS_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(2) input';
+const STORE_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(3) input';
+const OTHER_CHECKBOX_SELECTOR = '.jp-checkbox-answer__container:nth-child(4) input';
+
+export default class RecommendationsPage extends WpPage {
+	constructor( page ) {
+		const url = `${ siteUrl }/wp-admin/admin.php?page=jetpack#/recommendations`;
+		super( page, { expectedSelectors: [ '[class^=jp-recommendations-]' ], url } );
+	}
+
+	isUrlInSyncWithStepName( stepName ) {
+		const url = this.page.url();
+		return url.includes( stepName );
+	}
+
+	async areSiteTypeQuestionsVisible() {
+		const siteTypeQuestionsSelector = '.jp-recommendations-question__site-type-checkboxes';
+		return await this.waitForElementToBeVisible( siteTypeQuestionsSelector );
+	}
+
+	async checkPersonaSiteType() {
```

Comment:
There's a small typo in the method name.

Reply:
Good catch! Thank you, Kev.